contracts/truefi2/LoanToken2.sol

```diff
@@ -292,14 +284,18 @@ contract LoanToken2 is ILoanToken2, ERC20 {
     /**
      * @dev Get coupon value of this loan token in token
      * This assumes the loan will be paid back on time, with interest
-     * @param _balance number of LoanTokens to get value for
-     * @return coupon value of _balance LoanTokens in tokens
+     * @param _amount number of LoanTokens to get value for
+     * @return coupon value of _amount LoanTokens in tokens
      */
-    function value(uint256 _balance) external override view returns (uint256) {
-        if (_balance == 0) {
+    function value(uint256 _amount) external override view returns (uint256) {
+        if (_amount == 0) {
             return 0;
         }
 
+        if (status >= Status.Defaulted) {
```

Comment:
Does `>=` mean that status should be either `Defaulted` or `Liquidated` to enter if statement? If so, consider changing this to explicit comparison with those statuses for better readability.